src/components/Footer.tsx

```diff
@@ -0,0 +1,78 @@
+/* eslint-disable import/no-cycle */
+import React, { useContext } from 'react';
+import classNames from 'classnames';
+import { TodosContext } from '../TodoContext/TodoContext';
+import { Status } from '../types/Status';
+
+export const Footer: React.FC = () => {
+  const {
+    todos,
+    deleteCompletedTodos,
+    setQuery,
+    query,
+  } = useContext(TodosContext);
+
+  const unCompletedTodos = todos.filter(({ completed }) => !completed);
+
+  const completedTodos = todos.filter(({ completed }) => completed);
+
+  return (
+    <footer className="todoapp__footer" data-cy="Footer">
+      <span className="todo-count" data-cy="TodosCounter">
+        {`${unCompletedTodos.length} items left`}
+      </span>
+
+      <nav className="filter" data-cy="Filter">
+        <a
+          href="#/"
```

Comment:
Put `map` here, check prev comment in last review wich related to this code
```suggestion
       {...map)
```